airbyte-cdk/bulk/toolkits/load-low-code/src/test/kotlin/io/airbyte/cdk/load/lowcode/DeclarativeDestinationFactoryTest.kt

```diff
@@ -301,9 +317,309 @@ class DeclarativeDestinationFactoryTest {
         assertEquals(expectedOperations, actualOperations)
     }
 
+    @Test
+    internal fun `test dynamic operations with static destination objects`() {
+        val config =
+            Jsons.readTree("""{"api_id": "$VALID_API_ID", "api_token": "$VALID_API_TOKEN"}""")
```

Comment:
nit: This config does not align with the HttpRequester in the discover. Can we just set a variable `ANY_CONFIG` to be explicit about the config not being important in this test? If we want to test interpolation being populated properly, we can do it in another but I don't think this is necessarily needed

Reply:
yep fixing!